sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/TextSocketStreamSuite.scala

```diff
@@ -187,7 +187,7 @@ class TextSocketStreamSuite extends StreamTest with SharedSparkSession {
 
   test("non-boolean includeTimestamp") {
     val provider = new TextSocketSourceProvider
-    val params = Map("host" -> "localhost", "port" -> "1234", "includeTimestamp" -> "fasle")
+    val params = Map("host" -> "localhost", "port" -> "1234", "includeTimestamp" -> "false")
```

Comment:
Seems like it was testing a negative case. `fasle` seems right value to test.

Reply:
@HyukjinKwon ah ok..reverted the change for this one